src/utils/config/default_values.rs

```diff
@@ -1,7 +1,9 @@
 use zksync_ethers_rs::types::H160;
 
 pub const DEFAULT_L1_RPC_URL: &str = "http://localhost:8545";
+pub const DEFAULT_L1_CHAIN_ID: u64 = 9;
```

Comment:
Let's use Sepolia's chain ID
```suggestion
pub const DEFAULT_L1_CHAIN_ID: u64 = 1115111;
```

Reply:
Nvm

Reply:
We can, but we are using `RETH` as default:
```rust
pub const DEFAULT_L1_RPC_URL: &str = "http://localhost:8545";
```